core/src/credential.rs

```diff
@@ -38,41 +124,36 @@ impl CredentialType {
             "BankAccount" => Some(CredentialType::BankAccount),
             "PermanentResidentCard" => Some(CredentialType::PermanentResidentCard),
             "VerifiableCredential" => Some(CredentialType::Common),
+            "UniversityDegreeCredential" => Some(CredentialType::UniversityDegreeCredential),
+            "AlumniCredential" => Some(CredentialType::AlumniCredential),
             _ => None,
         }
     }
 }
 
-pub type CredentialSubject = std::collections::HashMap<String, serde_json::Value>;
-
-#[derive(Debug, serde::Serialize, serde::Deserialize, Clone)]
-pub struct VerifiableCredential {
-    #[serde(flatten)]
-    pub credential: Credential,
-    pub proof: crate::proof::DataIntegrityProof,
+impl std::fmt::Display for VerifiableCredential {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let vc = self.to_json_string().unwrap();
```

Comment:
Let me think about this, this would only fail if we had somehow constructed a Verifiable Credential or VP that was incorrect on validation. Which should be impossible. So I lean towards keeping the panic here, but let me think about it

Reply:
Done